primitives/fee-market/src/lib.rs

```diff
@@ -191,8 +187,7 @@ pub struct PriorRelayer<AccountId, BlockNumber, Balance> {
 	pub fee: Balance,
 	pub valid_range: Range<BlockNumber>,
 }
-
-impl<'a, AccountId, BlockNumber, Balance> PriorRelayer<AccountId, BlockNumber, Balance>
+impl<AccountId, BlockNumber, Balance> PriorRelayer<AccountId, BlockNumber, Balance>
 where
 	BlockNumber: sp_std::ops::Add<Output = BlockNumber> + Clone,
```

Comment:
Please help change this line to `BlockNumber: AtLeast32BitUnsigned + Copy` by the way.